requirements-dev.txt

```diff
@@ -10,7 +10,7 @@ jsonschema==3.2.0
 psutil==5.8.0
 lxml==4.6.3
 opencv-python==4.5.1.48
-spikeextractors==0.9.7
+spikeextractors @ git+https://github.com/spikeinterface/spikeextractors.git@master
```

Comment:
Is this necessary? It looks like the last change to SpikeGadgets was in March and the latest release was in July. Is there some feature this needs that I am missing?

Reply:
Mentioned why it's necessary in PR description - requires a debug that was quickly merged to master the other day: [se PR #662](https://github.com/SpikeInterface/spikeextractors/pull/662)

Reply:
@alejoe91 would it be possible to push a release of spikeextractors?

Reply:
Done, waiting for tests to pass now